### DISCUSSION TITLE
Rowy Support with Firebase - App Check

### DISCUSSION THREAD

**Discussion by KKimj** [AntlerVC/firetable#813] [Idea / Feature Request]
![화면 캡처 2022-09-16 145545](https://user-images.githubusercontent.com/40026920/190567126-190d70a7-15bd-4739-8db9-484e72bddd2c.png)

Hello!
Thanks your open source project, rowy!!

- I want to set up "Firebase - App Check" with rowy web.

- I use Rowy with GCP, Cloud Run.

- Is there a way to set up Rowy with Firebase App Check?

Thanks!
Have a nice day!
Best,

**Comment by harinij** [comment]
Hi @KKimj currently we don't support App check integration. Adding this to the idea/feature request queue.

**Reply by KKimj** [reply]
Hello @harinij 
Thanks for your reply!
We are checking where the unverified request is coming from.
Rowy is also counted in the unverfied request, so I'm telling you.

![image](https://user-images.githubusercontent.com/40026920/201008297-77b0835e-3f65-4831-8ec1-574bb3390b49.png)
